source/includes/kubernetes/_k8_persistentvolumeclaims.md

```diff
@@ -116,7 +116,7 @@ curl -X GET \
       "name": "cmc-staging-mysql",
       "namespace": "cmc-stg",
       "resourceVersion": "111495311",
-      "selfLink": "/api/v1/namespaces/cmc-stg/persistentvolumeclaims/cmc-staging-mysql",
+      "selfLink": "/api/api/v1/namespaces/cmc-stg/persistentvolumeclaims/cmc-staging-mysql",
```

Comment:
Double `/api` here

Reply:
Fixed in commit b72186371f3f9263f21b901b1608cb3131183861